intel-lustre-clients-on-centos/azuredeploy.json

```diff
@@ -0,0 +1,403 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "artifactsBaseUrl": {
+            "type": "string",
+            "defaultValue":"https://raw.githubusercontent.com/arsenvlad/azure-quickstart-templates/master/intel-lustre-clients-on-centos",
+            "metadata": {
+                "description": "Base URL of the solution template gallery package",
+                "artifactsBaseUrl": ""
+            }
+        },
+        "location": {
+            "type": "string",
+            "allowedValues": [
+                "westus",
```

Comment:
The reason I used the lower cased version is because createUiDefinition.json UI returns regions in lowercase. I will include the more readable form on top and the lowercase form on the bottom of the list to make sure that both options are covered.